content/en/user-guide/aws/dms/index.md

```diff
@@ -120,10 +120,12 @@ DMS is in a preview state on LocalStack and only supports some selected use case
 | RDS MariaDB        | Kinesis     | full-load, cdc  |
 | RDS MySQL          | Kinesis     | full-load, cdc  |
 | S3                 | Kinesis     | full-load, cdc  |
+| Aurora PostgreSQL  | Kinesis     | full-load, cdc  |
+| RDS PostgreSQL     | Kinesis     | full-load, cdc  |
 
 ## Serverless
 
-[DMS Serverless](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Serverless.html) can be used for the sources and targets that LocalStack already supports, and that are also [supported by AWS](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Serverless.Components.html#CHAP_Serverless.SupportedVersions).
+[DMS Serverless](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Serverless.html) can be used for the MariaDB and MySQL sources and kinesis target in LocalStack.
```

Comment:
Question: Should we update the table to indicate Serverless support instead? This feels not too clear. It might be easier to have 3 symbols: non supported by AWS/non applicable (S3 and externally managed databases), non supported by LocalStack (all RDS/Aurora Postgres), supported, or something like that?